Formula/s/synergy-core.rb

```diff
@@ -42,9 +43,13 @@ class SynergyCore < Formula
   end
 
   depends_on "cmake" => :build
+  # Synergy uses std::ranges::find, which seems to have been first available in Xcode 14.3.
+  # See https://developer.apple.com/documentation/xcode-release-notes/xcode-14_3-release-notes,
+  # which mentions implementation of "P0896R4 - The One Ranges Proposal".
+  depends_on xcode: ["14.3", :build]
```

Comment:
Does this actually need a full Xcode installation, or does it just need a compiler toolchain that understands `std::ranges`? I suspect it's the latter, in which case you should probably instead do something like
```ruby
fails_with :clang do
  build 1402
  cause "needs `std::ranges::find`"
end
```

You can probably also use
```ruby
on_macos do
  depends_on "llvm" => :build if DevelopmentTools.clang_build_version <= 1402
end
```
along with
```ruby
ENV.llvm_clang if OS.mac? && DevelopmentTools.clang_build_version <= 1402
```
in the `install` method.

Reply:
Thank you. I have implemented these changes.